attributes/default.rb

```diff
@@ -60,7 +62,17 @@
 default['cfncluster']['nvidia']['cuda_url'] = 'https://developer.nvidia.com/compute/cuda/10.0/Prod/local_installers/cuda_10.0.130_410.48_linux'
 # EFA
 default['cfncluster']['efa']['installer_url'] = 'https://s3-us-west-2.amazonaws.com/aws-efa-installer/aws-efa-installer-1.4.1.tar.gz'
-
+# DCV
+default['cfncluster']['dcv']['enabled'] = 'yes'
+default['cfncluster']['dcv']['version'] = '2019.0-7318'
+default['cfncluster']['dcv']['centos7'] = 'el7'
+default['cfncluster']['dcv']['url'] = "https://d1uj6qtbmh3dt5.cloudfront.net/2019.0/Servers/nice-dcv-#{node['cfncluster']['dcv']['version']}-#{node['cfncluster']['dcv']['centos7']}.tgz"
+default['cfncluster']['dcv']['server'] = "nice-dcv-server-2019.0.7318-1.#{node['cfncluster']['dcv']['centos7']}"
+default['cfncluster']['dcv']['xdcv'] = "nice-xdcv-2019.0.224-1.#{node['cfncluster']['dcv']['centos7']}"
```

Comment:
too many version strings here.
if ['dcv']['server'] and ['dcv']['xdcv'] are the only packages inside the node['cfncluster']['dcv']['centos7']}.tgz archive, we should avoid to specify them

Reply:
Unfortunately, they are not the only packages present in the tgz

Reply:
I meant that they are the only nice-dcv-server and nice-xdcv packages inside the archive, I don't see the need to specify their versions